test/connector/SchemeConnectorSpec.scala

```diff
@@ -98,15 +123,15 @@ object SchemeConnectorSpec {
       |  "items": [
       |    {
       |      "pstr": "00241768RH",
-      |      "declarationDate": "0001-01-01T00:00:00",
+      |      "relationshipStartDate": "0001-01-01T00:00:00",
       |      "schemeName": "THE AMDAIL PENSION SCHEME",
       |      "schemeOpenDate": "2006-04-05T00:00:00",
       |      "racDac": false,
       |      "policyNo": ""
```

Comment:
This too has changed to 'policyNumber', let's correct all for consistency

Reply:
Done